kit/popups/src/Sheets/UISheet/UISheet.tsx

```diff
@@ -214,11 +222,7 @@ function SheetContent({
                             ? { waitFor: scrollPanGestureHandlerRef }
                             : null)}
                     >
-                        <Animated.View
-                            onLayout={onSheetLayout}
-                            style={[style, cardSizeStyle]}
-                            pointerEvents="box-none"
```

Comment:
It disabled `PanGestureHandler` around the view (at least on Android), hence pan gestures didn't work on sheet